papis/commands/doctor.py

```diff
@@ -620,16 +620,45 @@ def biblatex_required_keys_check(doc: papis.document.Document) -> List[Error]:
 KEY_TYPE_CHECK_NAME = "key-type"
 
 
-def key_type_check(doc: papis.document.Document) -> List[Error]:
+def get_key_type_check_keys() -> Dict[str, type]:
     """
-    Check document keys have expected types.
+    Check the `doctor-key-type-check-keys` configuration entry for correctness.
 
     The :ref:`config-settings-doctor-key-type-check-keys` configuration entry
     defines a mapping of keys and their expected types. If the desired type is
     a list, the :ref:`config-settings-doctor-key-type-check-separator` setting
     can be used to split an existing string (and, similarly, if the desired type
     is a string, it can be used to join a list of items).
 
+    :returns: A dictionary mapping key names to types.
+    """
+    import builtins
+
+    key_type_check_keys = papis.config.getlist("doctor-key-type-check-keys")
+    processed_key_type_check_keys: Dict[str, type] = {}
+    # TODO: somehow i'm still getting a traceback here even when catching the exception
```

Comment:
Is this still relevant? Just ran a quick `papis doctor --checks key-type -a` and didn't get any tracebacks.

Reply:
Great catch, i forgot to check this file again!